File: src/main/java/org/manapart/jumpPads/ModBlocks.java
package org.manapart.jumpPads;

import net.minecraft.block.Block;
import net.minecraftforge.event.RegistryEvent;
import net.minecraftforge.eventbus.api.SubscribeEvent;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.registries.ForgeRegistries;

import static org.manapart.jumpPads.JumpPads.*;

@Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.MOD)
public class ModBlocks {

    @SubscribeEvent
    public static void registerBlocks(RegistryEvent.Register<Block> event) {
        if (!ForgeRegistries.BLOCKS.containsKey(JUMP_PAD.getRegistryName())) {
//            System.out.println("Registering blocks");
            ForgeRegistries.BLOCKS.register(JUMP_PAD);
            ForgeRegistries.BLOCKS.register(JUMP_PAD_2);
            ForgeRegistries.BLOCKS.register(JUMP_PAD_3);
        }
    }
}

File: src/main/java/org/manapart/jumpPads/ModBlocks.kt
package org.manapart.jumpPads

import net.minecraftforge.registries.DeferredRegister
import net.minecraftforge.registries.ForgeRegistries
import thedarkcolour.kotlinforforge.forge.registerObject

object ModBlocks {
    val REGISTRY = DeferredRegister.create(ForgeRegistries.BLOCKS, MODID)

    val JUMP_PAD by REGISTRY.registerObject("jumppad") { JumpPad(1.5f) }
    val JUMP_PAD_2 by REGISTRY.registerObject("jumppad_2") { JumpPad(3f) }
    val JUMP_PAD_3 by REGISTRY.registerObject("jumppad_3") { JumpPad(4f) }

}